pubsub/google/cloud/pubsub_v1/publisher/client.py

```diff
@@ -242,7 +243,14 @@ def publish(self, topic, data, **attrs):
             instance that conforms to Python Standard library's
             :class:`~concurrent.futures.Future` interface (but not an
             instance of that class).
+
+        Raises:
+            ValueError:
+                If called after publisher has been stopped
+                by a `stop()` method call.
         """
+        if self._is_stopped:
```

Comment:
We should have a lock around reads/writes to self._is_stopped yea?

Reply:
Hm-m. Pushed